x-pack/test/security_solution_cypress/es_archives/exceptions/mappings.json

```diff
@@ -0,0 +1,37 @@
+{
+  "type": "index",
+  "value": {
+    "aliases": {
+      "exceptions": {
+        "is_write_index": false
+      }
+    },
+    "index": "exceptions-0001",
+    "mappings": {
+      "properties": {
+        "@timestamp": {
+          "type": "date"
+        },
+        "agent": {
+          "properties": {
+            "name": {
+              "ignore_above": 1024,
+              "type": "keyword"
+            }
+          }
+        },
+        "user": {
+          "type": "nested",
+          "properties": {
+            "first": {
+              "type": "keyword"
+            },
+            "last": {
+              "type": "keyword"
+            }
+          }
+        }
+      }
+    }
+  }
+}
```

Comment:
fwiw, something I wished I had done before was add this to some of my tests and mappings:

https://github.com/elastic/kibana/blob/master/x-pack/test/functional/es_archives/filebeat/default/mappings.json#L5936

Which is a `"refresh_interval": "5s"` it reduces flake and speeds things up if you're pushing data from es archiver into it. Optional as I forget to add it. Without it, I think the default of tests are 1 minute. They might have changed the default for tests that I don't know about, but I don't know the answer to that either.